neqo-transport/src/fc.rs

```diff
@@ -347,10 +370,42 @@ impl ReceiverFlowControl<StreamId> {
         builder: &mut PacketBuilder,
         tokens: &mut Vec<RecoveryToken>,
         stats: &mut FrameStats,
+        now: Instant,
+        rtt: Duration,
     ) {
         if !self.frame_needed() {
             return;
         }
+
+        // Auto-tune max_active, i.e. the flow control window.
```

Comment:
This should be in a new function.  Then you can document it more cleanly and use early returns to make it neater.

Reply:
Extracted in f351b880382616cfbf028d9f87e69e777f042522.